DANE-for-SMTP-how-to.md

```diff
@@ -43,16 +43,50 @@ DANE is designed to work with any TLS service, not just email, but DANE for HTTP
 # Why use DANE for SMTP?
 The use of opportunistic TLS (via STARTTLS) is not without risks:
 * Because forcing the use of TLS for all mail servers would break backwards compatibility, SMTP uses opportunistic TLS (via STARTTLS) as a mechanism to enable secure transport between mail servers. However, the fact that STARTTLS is opportunistic means that the initial connection from one mail server to another always starts unencrypted making it vulnerable to man in the middle attacks. If a mail server does not offer the 'STARTTLS capability' during the SMTP handshake (because it was stripped by an attacker), transport of mail occurs over an unencrypted connection. 
-* By default mail servers do not validate the authenticity of another mail server's certificate; any random certificate is accepted (see [RFC 2487](https://tools.ietf.org/html/rfc2487). This was probably done because there is no user who can act on errors in case they occur. Unfortunately this default behavior introduces the risk of a man in the middle attack; an attacker can offer a false certificate enabling the attacker to decrypt encrypted traffic.
+* By default mail servers do not validate the authenticity of another mail server's certificate; any random certificate is accepted (see [RFC 3207](https://tools.ietf.org/html/rfc3207).
+    - It was unclear which CAs to trust to validate the certificate for a given destination.
+    - As a result of MX indirection, it was unclear which names to verify in the certificate.
```

Comment:
I'm unfamiliar with the concept of "MX indirection". Can you elaborate on this?

Reply:
Expanding the wording.